tests/unit/models/test_model_interfaces.py

```diff
@@ -144,6 +153,10 @@ def _3x_plus_10(x: tf.Tensor) -> tf.Tensor:
     return 3.0 * x + 10
 
 
+def _sin_10_x(x: tf.Tensor) -> tf.Tensor:
```

Comment:
@joelberkeley another test function that provides data easier to handle by GPs

Reply:
you ain't fooling me: that's not sin(10x)